packages/esm-patient-common-lib/src/time-helper.ts

```diff
@@ -11,5 +11,5 @@ export const convertTime12to24 = (time12h, timeFormat: amPm) => {
     hours = hours === '12' ? hours : parseInt(hours, 10) + 12;
   }
 
-  return [hours, minutes];
+  return [hours, minutes] as [number, number];
```

Comment:
The typing here is wrong. It returns `[ string | number, string ]`. I _actually_ think the intention here is to return `[ string, string ]` (so that we can represent midnight as `'00'` and 11 PM as `'23'`).

```suggestion
  return [hours, minutes];
```

Reply:
I'm also confused as to why this function is modified here.

Reply:
Right, because `time12h` is inferred as `any`, but clearly should be `string`, since we're expecting it to split on a character. Change that and the inferred type should be correct.

Reply:
@vasharma05 My point wasn't about what the function _should_ return, just what it actually returns. Casting to `as [number, number]` probably gets around Typescript reporting the issue, but it results in unsound types. In that case, the best thing to do would be to explicitly convert the string to a number, e.g.,

```js
return [Number(hours), Number(minutes)];
```